From b581868085890b237c93ef812934668bb5521315 Mon Sep 17 00:00:00 2001
From: Shuhao Wu <shuhao@shuhaowu.com>
Date: Mon, 5 Aug 2019 22:24:42 -0400
Subject: [PATCH 3/3] Increased limit on loot tracker SELECT

---
 .../http/service/loottracker/LootTrackerController.java         | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/http-service/src/main/java/net/runelite/http/service/loottracker/LootTrackerController.java b/http-service/src/main/java/net/runelite/http/service/loottracker/LootTrackerController.java
index 2c1bd7714..62fbdfc48 100644
--- a/http-service/src/main/java/net/runelite/http/service/loottracker/LootTrackerController.java
+++ b/http-service/src/main/java/net/runelite/http/service/loottracker/LootTrackerController.java
@@ -67,7 +67,7 @@ public class LootTrackerController
 	}
 
 	@GetMapping
-	public Collection<LootRecord> getLootRecords(HttpServletRequest request, HttpServletResponse response, @RequestParam(value = "count", defaultValue = "1024") int count, @RequestParam(value = "start", defaultValue = "0") int start) throws IOException
+	public Collection<LootRecord> getLootRecords(HttpServletRequest request, HttpServletResponse response, @RequestParam(value = "count", defaultValue = "16384") int count, @RequestParam(value = "start", defaultValue = "0") int start) throws IOException
 	{
 		SessionEntry e = auth.handle(request, response);
 		if (e == null)
-- 
2.22.0

